projects/demo/src/modules/app/pages.ts

```diff
@@ -302,6 +302,12 @@ export const pages: TuiDocPages = [
                 keywords: `dropdown, контекст, selection, выделение, выпадашка, дропдаун, Context`,
                 route: `/directives/dropdown-selection`,
             },
+            {
+                section: `Tools`,
```

Comment:
shouldn't this be in section "Customization"?

Reply:
fixed